app/views/ontologies/browser/browse.html.haml

```diff
@@ -1,5 +1,5 @@
 .browse-center
-  .browse-container
+  .browse-container{'data-controller': 'federation', 'data-federation-config-value': federated_portals.to_h.to_json}
```

Comment:
this data-controller does not need to be a this level, put it more near its usage, in `.browse-federation-input-chip-container` for example.